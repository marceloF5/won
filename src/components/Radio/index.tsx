import * as S from './styles'

const Radio = () => (
    <S.Wrapper>
        <h1>Radio</h1>
    </S.Wrapper>
)

export default Radio
