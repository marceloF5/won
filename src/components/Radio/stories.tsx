import { Story, Meta } from '@storybook/react/types-6-0'
import Radio from '.'

export default {
    title: 'Radio',
    component: Radio
} as Meta

export const Default: Story = () => <Radio />
