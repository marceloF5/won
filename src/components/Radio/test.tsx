import { render, screen } from '@testing-library/react'

import Radio from '.'

describe('<Radio />', () => {
    it('should render the heading', () => {
        const { container } = render(<Radio />)

        expect(
            screen.getByRole('heading', { name: /Radio/i })
        ).toBeInTheDocument()

        expect(container.firstChild).toMatchSnapshot()
    })
})
