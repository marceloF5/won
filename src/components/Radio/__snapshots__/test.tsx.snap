// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Radio
  </h1>
</main>
`
